seametrics/softmin_score/softmin_score.py

```diff
@@ -0,0 +1,196 @@
+import torch
+from tqdm import tqdm
+from transformers import MaskFormerForInstanceSegmentation, MaskFormerImageProcessor
+import cv2
+import numpy as np
+import torch.nn.functional as NNF
+from fiftyone import ViewField as F
+import fiftyone as fo
+from cleanlab.segmentation.rank import get_label_quality_scores, issues_from_scores 
+from cleanlab.segmentation.filter import find_label_issues 
+import time
+from PIL import Image
+import albumentations as A
+
+DEVICE = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+DATASET_NAME = "TRAIN_PANOPTIC_DATASET"
+MODEL_PATH =  "/mnt/hdd/maryam/models/model_panoptik_27k_saved_100epochs/last_model"
```

Comment:
can we fetch the model from HF instead of a local path?

Reply:
Done